code/modules/events/blob.dm

```diff
@@ -14,11 +14,10 @@
 	var/mob/C
 	if(candidates.len)
 		C = pick(candidates)
-		Blob = new /obj/effect/blob/core(T, 200)
+		Blob = new /obj/effect/blob/core(T, 200, null, 2, 0, 1)

```

Comment:
> create_overmind returns early the first time it is called

No, it returns early for 5 minutes *after* the first time it is called.

> I did not trust that in the few seconds around spawn, a Life() call wouldn't call create_overmind() again

It will, but `create_overmind`'s internal cooldown will prevent it from doing anything.

> The method that I am currently using completely blocks create_overmind from being called automatically, but still allows it to be called manually to put the right player into the blob.

You should **not** be calling it manually. It's designed to be called internally, with a pre-chosen client passed in through `New`.